config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```
@@ -22,6 +22,80 @@ periodics:
       args:
       - -c
       - "cd groups && make run -- --confirm"
+- name: ci-k8sio-audit
+  interval: 6h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  max_concurrency: 1
+  annotations:
+    testgrid-dashboards: wg-k8s-infra-k8sio
+    testgrid-alert-email: hh@ii.coop
+    testgrid-num-failures-to-alert: '100'
+  extra_refs:
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: master
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  spec:
+    serviceAccountName: k8s-infra-gcp-auditor
+    containers:
+    - name: groups
+      image: gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f
+      command:
+      - bash
+      args:
+      - -c
+      - |
+        echo "Some initial debuging to figure out what it looks like here" >&2
+        pwd
+        ls
+        echo "Ensure gcloud creds are working" >&2
+        gcloud config list
+        echo -n "Calculate github user from token: " >&2
+        GH_TOKEN=$(cat /etc/github-token/oauth)
+        GH_USER=$(curl -H "Authorization: token $GH_TOKEN" "https://api.github.com/user" 2>/dev/null | sed -n "s/\s\+\"login\": \"\(.*\)\",/\1/p")
+        FORK_GH_BRANCH=autoaudit-${PROW_INSTANCE_NAME:-prow}
+        echo "$GH_USER" >&2
+        echo "Running Audit Script to dump GCP configuration to yaml" >&2
+        cd ../audit
+        ./audit-gcp.sh
+
+        echo "Generate pr-creator binary from k/test-infra/robots" >&2
+        pushd ../test-infra
+        go build -o ../k8s.io/pr-creator robots/pr-creator/main.go
+        popd
+
+        echo "Prepare commit for possible PR"
+        # Values pulled from config/prow/autobump-config.yaml
+        # and logic from prow/cmd/autobump.sh
+        git config user.name "Kubernetes Prow Robot"
+        git config user.email "k8s.ci.robot@gmail.com"
+        git add --all -u
+        git commit -m "audit: update as of $(date +%Y-%m-%d)"
+        echo -e "Pushing commit to github.com/${GH_USER}/${FORK_GH_REPO}:..." >&2
+        git push -f "https://${GH_USER}:@github.com/${GH_USER}/${FORK_GH_REPO}" "HEAD:${FORK_GH_BRANCH}}" 2>/dev/null
```

I think this is going to require a token?
```suggestion
        git push -f "https://${GH_USER}:${GH_TOKEN}@github.com/${GH_USER}/${FORK_GH_REPO}" "HEAD:${FORK_GH_BRANCH}}" 2>/dev/null
```